packages/space-opera/src/components/mobile_view/mobile_view.ts

```diff
@@ -64,14 +66,12 @@ export class MobileView extends LitElement {
   @internalProperty() sessionUrl = getSessionUrl(this.pipeId, this.sessionId);
   @internalProperty() sessionOs = getMobileOperatingSystem();
 
-  updateState(snippet: any, envChanged: boolean) {
-    // The partialState env link correspondes to the editor's link.
-    if (envChanged) {
-      snippet.config.environmentImage = undefined;
-    } else if (this.config.environmentImage) {
-      snippet.config.environmentImage = this.config.environmentImage;
-    }
+  get needIosSrc(): boolean {
+    return this.sessionOs === 'iOS' && this.iosUrl.length <= 1;
```

Comment:
Why `<= 1`?

Reply:
just a metric for determining whether or not there exists a url yet by the string,

Reply:
I guess I just figured the check would be for `< 1` or `=== 0`. Is there some special 1-character case?